src/MToken.sol

```diff
@@ -16,12 +17,29 @@ import { IMToken } from "./interfaces/IMToken.sol";
 import { ContinuousIndexing } from "./abstract/ContinuousIndexing.sol";
 import { ContinuousIndexingMath } from "./libs/ContinuousIndexingMath.sol";
 
+abstract contract MTokenStorageLayout {
+    /// @custom:storage-location erc7201:m0.storage.MToken
+    struct MTokenStorage {
+        bytes32 name;
+        bytes32 symbol;
+    }
+
+    // keccak256(abi.encode(uint256(keccak256("m0.storage.MToken")) - 1)) & ~bytes32(uint256(0xff))
+    bytes32 private constant _M_TOKEN_STORAGE_SLOT = 0x93466981a55ce4da70da67506024f05bd15faf7fdab908fab669c24c9fb04d00;
```

Comment:
Has the hash been updated?
I get `0xeb521740570240fc3b71275bd090460409d92adf7cec95ba2df10066ee73c700` when calling the code above.